### PR TITLE
Homework #14 (unit test)

### PR DESCRIPTION
**Pull request by vetalbunyak**

### CODE REVIEW

File: src/test/java/ithillel/ua/leetCodeProblemsTest.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    @Test
+    public void testNumIdenticalPairs() {
+        int [] nums = new int[] {1,1,1,1};
+        int expected = 6;
+        int actual = test.numIdenticalPairs(nums);
+        Assert.assertEquals(expected,actual);
+    }
+
+    @Test
+    public void TestMaxNumberOfBalloons() {
```
Comment (mkharkhuta) [inline src/test/java/ithillel/ua/leetCodeProblemsTest.java:26]:
```suggestion
    public void testMaxNumberOfBalloons() {
```

**Review by mkharkhuta — COMMENTED** [review]
Good, but please, use Given-When-Then rule for test naming. Also real tests scenarios should contain positive, negative and edge cases 
[](https://medium.com/@gitaeklee/given-when-then-junit-test-ba49564303e7)
